inference-engine/samples/benchmark_app/utils.cpp

```diff
@@ -221,11 +219,19 @@ std::map<std::string, std::vector<float>> parseScaleOrMean(const std::string& sc
 
 #ifdef USE_OPENCV
 void dump_config(const std::string& filename, const std::map<std::string, std::map<std::string, std::string>>& config) {
+    auto plugin_to_opencv_format = [](const std::string& str) -> std::string {
+        std::string new_str(str);
+        auto pos = new_str.find(".");
+        if (pos != std::string::npos) {
+            new_str.replace(pos, 1, "_");
```

Comment:
I propose to add check somewhere that if the original device name contains "_" then config can't be dumped.  Also maybe there is a sense to add WARNING that the device name is changed due to limitations.